src/parser/transformer.cpp

```diff
@@ -498,34 +498,48 @@ unique_ptr<ParsedExpression> Transformer::transformListOperatorExpression(
     CypherParser::OC_ListOperatorExpressionContext& ctx,
     unique_ptr<ParsedExpression> propertyExpression) {
     auto rawExpression = propertyExpression->getRawName() + " " + ctx.getText();
-
-    if (ctx.children[1]->getText() == ":" || ctx.children[2]->getText() == ":") {
+    if (ctx.kU_ListSliceOperatorExpression()) {
```

Comment:
I'll approve this PR but you can make the following change before merging. The general principal of transformer is to follow grammar. Since you have split grammar into different rules, this part need to change accordingly.

```
unique_ptr<ParsedExpression> listOperator;
if (ctx.SliceOperator)
     listOperator = transformSlice ...
else if
     listOperator = transformExtract...
if ctx.oC_ListOperatorExpression 
        return transformListOperatorExpression(*ctx.oC_ListOperatorExpression(), move(listOperator));
return listOperator;
```

Reply:
Done